client/src/FormFrontend/FormFrontend.js

```diff
@@ -297,6 +296,42 @@ class FormFrontend extends React.Component {
             </Fragment>
           );
         }
+        case "form_faq":
+          return (
+            <>
+              <div style={{ fontWeight: 700 }}>
+                {text_maker("form_frontend_faq_note")}
+                {required_asterisk}
+              </div>
+              <FancyUL>
+                {_.map(field_info.faq_content, ([q, a], idx) => {
+                  return (
+                    <div key={idx}>
+                      <div style={{ fontWeight: "bold" }}>{text_maker(q)}</div>
+                      <TM k={a} />
```

Comment:
The Q&A text itself should belong to the template on the server side. It's a leak for the backend to even know that the `text_maker` ecosystem exists on the frontend; it would add one more place where maintenance/deployment of both are, indirectly, coupled.

The two _are_ coupled in a number of ways. Ideally that'd be strictly limited to what explicitly is communicated through the backend's API. It largely is, _but_ the client does need to correctly implement behaviour for all possible form types or else it risks failing the backend's validation. I think that's unavoidable though, since the responsibilities for display and behaviour clearly belong to the client. The content of the form templates shouldn't live in both places though.

Reply:
Possibly subtle, but it's fine for text keys like `"form_frontend_faq_note"` to belong to the client because that's something that is an implementation detail of the client's generic `"form_faq"` type field, rather than the content of any specific template owned by the backend.